src/java/org/apache/poi/common/usermodel/fonts/package-info.java

```diff
@@ -0,0 +1,4 @@
+@NonNullApi
```

Comment:
Mark every return value and parameter as being or needing to be non-null unless annotated otherwise. I don't know if this is desired or feasible, but it's what most projects choose as the base. "non-null by default, explicitly nullable via `@Nullable`.